app/src/main/java/com/example/hungrywolfs/adapters/DetailsAdapter.kt

```diff
@@ -0,0 +1,46 @@
+package com.example.hungrywolfs.adapters
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.TextView
+import androidx.recyclerview.widget.RecyclerView
+import com.example.hungrywolfs.R
+import com.example.hungrywolfs.network.MealDetailsInfo
+
+class DetailsAdapter :
+    RecyclerView.Adapter<DetailsAdapter.DetailsViewHolder>() {
+    private val mealDetails: MutableList<MealDetailsInfo> = mutableListOf()
```

Comment:
What exactly does this adapter? Does it display a list of meals? A list of tags? Both?

Reply:
It displays both

Reply:
I verified and I forgot to delete mealDetails list and it's associated methods from the previous version.